Jenkinsfile_CNP

```diff
@@ -80,16 +85,19 @@ def archiveCoreCaseDataDefinitions(env) {
  *
  * @param env target environment
  */
-def uploadCoreCaseDataDefinitions(env) {
+def uploadCoreCaseDataDefinitions(env, useShutteredCaseDef) {
   dir("${WORKSPACE}/bin"){
     if("preview" == env){
-      ccdDefinitionFilePath="${WORKSPACE}/build/ccd-release-config/ccd-fpl-preview-${CHANGE_ID}-toggle-on.xlsx"
+      ccdDefinitionFilePath = "${WORKSPACE}/build/ccd-release-config/ccd-fpl-preview-${CHANGE_ID}-toggle-on.xlsx"
       sh """
         eval \$(./variables/load-preview-environment-variables.sh ${CHANGE_ID})
         ${WORKSPACE}/fpla-docker/bin/utils/ccd-import-definition.sh ${ccdDefinitionFilePath}
       """
     } else {
-      ccdDefinitionFilePath="${WORKSPACE}/build/ccd-release-config/ccd-fpl-${env}.xlsx"
+      ccdDefinitionFilePath = "prod" == env && useShutteredCaseDef == '1'
```

Comment:
do we need the prod condition here, if we remove it then we can easily test on other envs as well, as long as the secret pulled is per env

Reply:
you're right - gonna bin it off